### DISCUSSION TITLE
Xdrip possibly doesn't work with newer Dexcom ONE transmitters

### DISCUSSION THREAD

**Discussion by powermos** [NightscoutFoundation/xDrip#2556] [General]
I heard from multiple sources that Dexcom did something to newer ONE transmitters (serial starting with "C") to prevent third party apps from getting data from the transmitter. Is this true? I also saw that the Anubis team is trying to get hold of one of these transmitters, to investigate.

**Comment by Navid200** [comment]
xDrip cannot currently be used with transmitters whose serial number starts with C.

**Reply by ChrisLJames** [reply]
Do you know if anybody is working on a fix to allow Xdrip to work on any Dexcom One sensor, it's difficult to know what serial number will arrive especially if there on an NHS prescription. I'm currently using Xdrip 2022.08.03 Libre1 via a Miaomiao but I can't even see any option for the Dexcom ONE only G5/G6

**Reply by ChrisLJames** [reply]
Just upgraded to the Nov release, I can see Dex1 option now but I guess the issue of serial numbers is still outstanding

**Reply by Navid200** [reply]
The Dexcom One option has been there for a quite a while.  It's just using it as if it was a G6.  And the menus have been recently updated to reflect that on a few of them.

No feature can be added immediately.  We need samples.  Currently, G7 and Dexcom One with serial number starting with C.  
And until it is supported, there is nothing we can tell you.  It makes no sense to make any predictions as on when it will be available considering we are all volunteers.  It will serve no purpose.

**Reply by ChrisLJames** [reply]
Thanks for the reply. I just wanted to say I can't say how much I appreciate how much work you guys put into developing and improving Xdrip, I've been using Xdrip for many years now and it's really brilliant, I think some of the Apps provided by some the manufactures should follow your lead.

**Comment by Navid200** [comment]
xDrip can work with these transmitters as of January 2023.

**Comment by PeterBaetschi** [comment]
Dear friends 
1. My Samsung Galxy 20fe 5G worked very well with Dexcom G6 using my Galaxy Watch 5pro as wear collector. That means my Galaxy Watch 5pro worked stand alone and read the Dexcom G6
2. My Samsung Galaxy 20fe 5G with xdrip works very well with dexcom G7/one. But when I force the Samsmg Galaxy phone to use the wear collector, it doesn't work.

-> for reading the Dexom G6 with the Watch as wear collector, I used an old xdrip version, that reads out every 9 minutes.

Now I look for this old xdrip apk with the 9 minutes rhythm. But I can not find this apk anymore.

Can someone send my a link to this old xdrip apk?

Thank you very much for your help!
